### DISCUSSION TITLE
公開Webミーティングの企画

### DISCUSSION THREAD

**Discussion by keiji** [cocoa-mhlw/cocoa#1168] [Ideas]
#1144 に関連して、COCOAのオープンソースコミュニティとして公開Webミーティングを考えています。

複数回に分けて1回30分から1時間程度、ゲストをお呼びして公開Web会議形式で進める。その時間に参加してくれた人が意見を言える時間を作るイメージです。

ベースはぼくが #1144 に投稿した課題を見ながらを考えていますが、ゲストや参加者に新しい課題を提示してもらっても良いと思います。

主題としては、オープンソースコミュニティとして、行政として、「次」に備えてどうするのが良いかを考えていく流れです。
ここで言う「次」は、オープンソースコミュニティと行政との協働を考えていますが、もちろん次のパンデミックに備える形でも良いと思います（ここはあまり限定せず進めたいと考えています）。

 * COCOAとオープンソースの取り組み
     * COCOAログチェッカーとか周辺の取り組み
     * 画面の変更に伴う行政内での調整
 * 接触確認API
     * 他に選択肢はなかったのか（GPSとか基地局情報とか、シンガポールのような独自デバイス
     * APIバージョンアップの課題（COCOAとして）
     * パラメーターの設定
 * System of SystemとしてのCOCOA（HER-SYSとの関連）
 * COCOAとワクチン接種証明アプリ。何が違うのか（体制は？　意思決定は？）

基本的にぼくは技術的なところの解像度は高いのですが、それ以外になると（当然ながら）ふらついて企画としてまとまりにくいので、ご意見をいただけると助かります。

また、HER-SYSや体制に関しては、ぼくより詳しい人が行政内にたくさんいるので、場合によっては行政官の方をお呼びしてお話しすることも考えています。

**Comment by keiji** [comment]
## 「COCOAとそれを取り巻くツール開発について」

COCOAだけではなく、不十分と思われるところを補うツールやサービスが立ち上がりました。
ツールやサービスを作成された方をゲストにお呼びして、どんな思いでサービスを立ち上げたか。サービス運営上の苦労を伺いたい。
それと同時に「なぜCOCOAが同じ事をできなかったのか」についても触れる必要があります。「できない事情があったからしょうがない」ではなく、その事情をどのように解消するか。突き詰めていくことで今後の課題が見えてくると考えます。

### ゲスト
@ktansai
@kvaluation 

### 日時
10月29日（土）　13時から14時

~~※ こちら枠の確保のために設定しています。URL等は変更になる可能性があります。~~ 確定しました

COCOA 公開Webミートアップ
10月 29日 (土曜日) · 午後1:00～2:00
Google Meet の参加に必要な情報
ビデオ通話のリンク: https://meet.google.com/qtm-avpb-hht

※ 本ミートアップについては現時点で録画を公開する予定はなく、議事録の公開のみ予定しています。

### 資料（参考含む

 * [COCOA公開Webミーティング-20221029.pdf](https://github.com/cocoa-mhlw/cocoa/files/9892555/COCOA.Web.-20221029.pdf)
 * 接触確認アプリの活用事例等
   * https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/cocoa_00138.html#h2_free9
 * 接触確認アプリ「COCOA」 感染拡大でウェブサイト利用者が増加
   * https://www3.nhk.or.jp/news/html/20220813/k10013769121000.html
 * 新cocoaログチェッカーの使い方（接触確認アプリ） - いなわくTV
   * https://youtu.be/fJRr0oJ_fOw

#### 基準値未満の接触画面（v2.0.0に搭載しようとしていた画面
 * https://github.com/cocoa-mhlw/cocoa/issues/823

**Reply by fujihalab** [reply]
@keiji さま，こちらのwebミーティングの件，ぜひ私も拝聴したく思っております．
もしよろしければ参加方法の詳細が決まりましたらご連絡いただけますと幸いです．
どうぞよろしくお願いいたします．@fujihalab

**Reply by keiji** [reply]
@fujihalab 時間と場所（URL）が確定したのでお知らせします！

**Reply by kvaluation** [reply]
資料（鈴木健治）

-  広範な接触による行動変容
  [Googleスプレッドシート](https://docs.google.com/presentation/d/1NZ96x8gKeJDaNc0cK_DeH7CxmF7G-tW-IabsKqDAxUg/edit?usp=sharing)
  [pdf](https://github.com/cocoa-mhlw/cocoa/files/9893129/20221029.pdf)

**Reply by keiji** [reply]
書き起こしのスクリプトです（調整前です）。
https://github.com/cocoa-mhlw/cocoa/blob/cocoa_meeting_scripts/documents/static/cocoa_meeting/meetup_2022-10-29_13_00.md

いただいたコメントです（ただし、ネットワーク回線が良くないみたいな話しかしていない）。
https://drive.google.com/file/d/1Ku3S7CS-pFEVPXmt4R2C-r3fMTa_uRS1/view?usp=share_link

動画です（公開場所は変更する可能性があるので直リンクはおすすめしません）
https://drive.google.com/file/d/1kjRNDqQGrH-3ohWVDEqRweIyl7gsnaoD/view?usp=share_link

**Comment by keiji** [comment]
## 「COCOAのENv2バージョンアップに関する取り組み」

### 概要

COCOA v2.0.0に含まれるもっとも大きな変更として、接触通知APIのENv2へのバージョンアップがあります。
Google/AppleがENv2をリリースしたのが2020年8月、ENv2対応のCOCOAがリリースされたのが2022年4月と、対応まで2年弱の間が空いています。

https://developers.google.com/android/exposure-notifications/release-notes#v16_august_2020
https://github.com/cocoa-mhlw/cocoa/releases/tag/v2.0.0

なぜここまで対応に時間がかかったのか。大きな課題だと考えてます。

このことについて、COCOAの事情、解決した手段を頭出しした上で、今後はどのようにすればよいかを考えていきたい。

たとえば、COCOAの場合はマルチプラットフォーム開発環境であるXamarinを使っていて、Xamarin対応の接触通知APIライブラリでENv2対応のものが存在しなかった。と言うことが1つの要因です。一方、事実としてCOCOAはXamarinのままENv2対応ができたわけで、現実的に不可能だったわけでもありません。

対応が遅くなった要因は、もっと別にあるのではと有山個人としては感じていて、そこを深掘りしていきたいと考えています。

この件についてはCOCOAに限定せず、オープンソースプロジェクトのコントリビューターの方をゲストにお呼びして課題と解決策を探りたいと考えています。

### ゲスト
@kishikawakatsumi

### 日時

COCOA 公開Webミートアップ
10月 25日 (火曜日) · 午後7:30～8:30
Google Meet の参加に必要な情報
ビデオ通話のリンク: https://meet.google.com/rbp-syvb-kvb

※ 内容は録画の上、後日公開する予定です。

### 資料（参考含む

 * [COCOA公開Webミーティング-20221025.pdf](https://github.com/cocoa-mhlw/cocoa/files/9858936/COCOA.Web.-20221025.pdf) （2022/10/25 18:21更新）
 * ExposureNotification APIについて開発チームを交えて開催した勉強会の資料
     * [ExposureNotification API勉強会.pdf](https://github.com/cocoa-mhlw/cocoa/files/9848684/ExposureNotification.API.pdf)
 * ENv2コード取り込み前に開発チームと意識あわせをしたときの資料
     * [ENv2コード取り込みについて.pdf](https://github.com/cocoa-mhlw/cocoa/files/9848685/ENv2.pdf)

**Reply by keiji** [reply]
本日はご参加いただきありがとうございました。よろしければフィードバックのアンケートにご協力ください。
https://docs.google.com/forms/d/e/1FAIpQLSfNnMCmWsVHy4hgLCZAGNEFImJyBti2kK3eJuX018FHeJR09A/viewform

動画は後日の公開となります。公開されたらまたご報告します。

**Reply by keiji** [reply]
書き起こしのスクリプトです（調整前です）。
https://github.com/cocoa-mhlw/cocoa/blob/cocoa_meeting_scripts/documents/static/cocoa_meeting/meetup_2022-10-25_19_29.md

いただいたコメントです。
https://drive.google.com/file/d/1x7b-nHEpoFUaR9ZFaegXF2jithsagiEf/view?usp=sharing

動画です（公開場所は変更する可能性があるので直リンクはおすすめしません）
https://drive.google.com/file/d/19NLD3dP7Mrq1TY68pPhymv5zyqPsF-b5/view?usp=sharing

**Comment by keiji** [comment]
## 「ワクチン接種証明アプリとCOCOA」

さまざまな課題があるCOCOAですが、デジタル庁がリリースしている「ワクチン接種証明アプリ」はわかりやすく機能していて、利用者からの評判も良いと個人的に感じています。

行政が開発したこれらの2つのアプリ、成否を分けた要因はなんなのか。

デジタル庁の行政官をゲストに迎えて、「ワクチン接種証明アプリ」の開発や意思決定の体制などを伺いながら「なぜそれがCOCOAでできなかったのか」、課題と解決策を探りたいと考えています。

## ゲスト
デジタル庁　国民向けサービスグループ　参事官補佐
市川 壱石 氏

## 日時
COCOA 公開Webミートアップ
10月 26日 (水曜日) · 午前11:00～午後12:00
Google Meet の参加に必要な情報
ビデオ通話のリンク: https://meet.google.com/arn-jfqu-xqv

※ 内容は録画の上、後日公開する予定です。

### 資料（参考含む

 * [COCOA公開Webミーティング-20221026.pdf](https://github.com/cocoa-mhlw/cocoa/files/9864837/COCOA.Web.-20221026.pdf) - 一応、話を始めるきっかけとして資料は作ったけど、ワクチン接種証明アプリについては本当に分からないので時間の中で聞いていきたいです。
 * ワクチン接種証明に挑む“官民ごちゃまぜ”チーム。一大プロジェクトの舞台裏
     * https://www.businessinsider.jp/post-248275

**Reply by keiji** [reply]
Technical specifications for EU Digital COVID Certificates - Volumes 1-5
https://health.ec.europa.eu/publications/technical-specifications-eu-digital-covid-certificates-volumes-1-5_en

**Reply by keiji** [reply]
本日はご参加いただきありがとうございました。よろしければフィードバックのアンケートにご協力ください。
https://docs.google.com/forms/d/e/1FAIpQLScWV5FMudy59F0MbQGZXNv4S68rS08ADJs7rFQMF1Sc_swwuQ/viewform

動画は後日の公開となります。公開されたらまたご報告します。

**Reply by keiji** [reply]
書き起こしのスクリプトです（調整前です）。
https://github.com/cocoa-mhlw/cocoa/blob/cocoa_meeting_scripts/documents/static/cocoa_meeting/meetup_2022-10-26_11_00.md

いただいたコメントです。
https://drive.google.com/file/d/1zE29SlsE9CvId1sbVu-xQdgnWO_kfdJI/view?usp=sharing

動画です（公開場所は変更する可能性があるので直リンクはおすすめしません）
https://drive.google.com/file/d/1p2y1nEHhkWxX83OppaUnps3-pOgosP-b/view?usp=sharing

**Comment by keiji** [comment]
## 「接触確認アプリを考える（コミュニティとして）」

今回、接触追跡（Contact Tracing）を実現する手段としてGoogle / Appleの「接触通知API」を採用しましたが、そもそも他に選択肢はなかったのか。そこからもう一度考えてみたいです。

位置情報の推定にあたってBLEに加えて、GPSや基地局情報（これらは接触通知APIでは認められていない手段です）や、それらの組み合わせることで、精度を上げることは技術的には可能です。また、シンガポールのように独自のデバイスを配布して接触を追跡をしている事例もあります。

一方、そもそも感染症対策をしている・していないをアプリから判定できない前提で、接触通知をどのような位置づけにすべきか。
たとえば日本では「接触通知を受けた人は公費負担で検査ができる」という取り扱いにしたため陽性者数が増えたときに接触通知を受ける人が増え、検査を受けたい人を増やして、保健所や医療機関の負担になったものと個人的には認識しています。

COCOAはどのような温度感で接触通知をするのが良かったのでしょうか。
そして、それを誰が決められるのでしょうか。厚生労働省でしょうか。または有識者の会議体でしょうか。

他国での接触通知アプリの運用事例についても改めて確認したい。
ドイツやイギリスの接触確認アプリの運用。前述のシンガポールの独自デバイス方式。

## ゲスト
@moonmile 

## 日時
COCOA 公開Webミーティング
11月 4日 (金曜日) · 午後7:00～~~8:00~~ 8:30 ← 時間拡大しました
Google Meet の参加に必要な情報
ビデオ通話のリンク: https://meet.google.com/smf-txtr-iug

※ 内容は録画の上、後日公開する予定です。

## 資料（参考含む
 * [COCOA公開Webミーティング-20221104.pdf](https://github.com/cocoa-mhlw/cocoa/files/9936512/COCOA.Web.-20221104.pdf)

**Reply by keiji** [reply]
「ワクチン接種証明アプリとCOCOA」について言及いただいたのですが、取り上げ方としてはこちらの方がいいかもしれないと思いました。念のためぶら下げておきます。

> どなたか「COCOAのほうは有識者会議（特に医療系）が継続されていたか？」を確認して頂けますか？
> ワクチン証明アプリのほうは単なるツールなのですが、COCOA のほうは公衆衛生に関わるものなので、継続が必要ではなかったかと考えています。
https://twitter.com/moonmile/status/1585074940436688896

**Reply by keiji** [reply]
このコメントに通じるものがある。
https://github.com/cocoa-mhlw/cocoa/issues/1144#issuecomment-1249995938

**Reply by moonmile** [reply]
プレゼン資料です。以前のものに加筆してあります。
これをベースにしてざっと1時間弱ぐらいで話して、後半30分はフリーの予定です。
（当日、時間の都合がつかない方用ということで）

接触確認アプリCOCOAの技術解説
https://1drv.ms/p/s!AmXmBbuizQkXgsZ5cqJszVAFCU2k6Q?e=OuxBZO

**Reply by keiji** [reply]
書き起こしのスクリプトです（調整前です）。
https://github.com/cocoa-mhlw/cocoa/blob/cocoa_meeting_scripts/documents/static/cocoa_meeting/meetup_2022-11-04_19_00.md

いただいたコメントです。
https://drive.google.com/file/d/1xJ30uNeqc7m4jZ0_MrunBOGNTYXbsNqp/view?usp=share_link

動画です（公開場所は変更する可能性があるので直リンクはおすすめしません）
https://drive.google.com/file/d/1Z51SuCnVytFhdkVbXprUhMiqg1p-tTSK/view?usp=share_link

**Comment by keiji** [comment]
## アンカーをまとめる場所

初の試みで、フリートークみたいになってしまっても建設的な話にならなさそうなので、COCOAに関する話題に戻れるアンカーを置いておく。話がグダグダになりそうだったらアンカーから選んで話を戻す。

**Reply by keiji** [reply]
PUBLIC DIGITAL （パブリック・デジタル）
アンドリュー・グリーンウェイ
ベン・テレット
マイク・ブラッケン
トム・ルースモア　著
岩嵜博論　監訳
川﨑千歳　訳
英治出版
http://www.eijipress.co.jp/book/book.php?epcode=2317

"接触追跡アプリはほとんどが当初の期待に応えることができなかった。接触追跡アプリの効果を100万人都市でシミュレーションしたオックスフォード大学の研究によると、ウイルスを抑え込むには推定80%のスマートフォン・ユーザーがアプリを使用する必要がある。だが、2021年の夏時点で普及率がこのレベルに達した国はないと見られている。シンガポールではほとんどの労働者が法的に義務付けられているにもかかわらず、政府がリリースした接触追跡アプリTraceTogether（トレース・トゥギャザー）をダウンロードしたのは、2020年6月末時点で3人に1人だけだった[1]。英国では廃棄されたバージョンのアプリに少なくとも3500万ポンドが支出され[2]、オーストラリアのアプリに至っては、2021年1月時点の検出済み接触事例の総数がたったの17件で、1件につき40万豪ドルを連邦政府が負担した計算になった[3]。"

[1] https://www.tracetogether.gov.sg/
[2] https://www.digitalhealth.net/2020/09/total-cost-of-nhs-contact-tracing-app-set-to-top-35-million/
[3] https://www.zdnet.com/article/australias-covidsafe-costs-au100000-per-month-to-keep-running/

**Reply by keiji** [reply]
COCOAトラブルの要因　内情知る元幹部が明かす「お役所体質」
https://mainichi.jp/articles/20221013/k00/00m/040/406000c

「お役ご免」COCOAの教訓から学ぶべきことは
https://mainichi.jp/articles/20221018/k00/00m/040/169000c

**Reply by keiji** [reply]
ここで示されている再発防止策をどの程度、講じることができているかも確認していきたい。

COCOA不具合調査・再発防止策検討チーム
https://www.mhlw.go.jp/stf/shingi/other-soumu_030416.html

**Reply by keiji** [reply]
複数のリポジトリがいい感じに運営されている国土地理院のアカウント。
GitHubアカウントは1つだけだけど、何人くらいで回してるんだろう。

国土地理院情報普及課公式GitHubアカウント
https://github.com/gsi-cyberjapan

**Reply by keiji** [reply]
新型コロナウイルス感染症対策 テックチーム Anti-Covid-19 Tech Team
https://cio.go.jp/techteam

接触確認アプリ及び関連システム仕様書
https://cio.go.jp/sites/default/files/uploads/documents/techteam_20200526_01.pdf

> 1. 目的
> 本アプリケーション（以下「本アプリ」と記述）は、スマートフォンの近接通信機構（Bluetooth）を利用し、人と人との接触したことを検知、記録する。新型コロナウイルス感染症の陽性診断が確定した者（以下「陽性者」と記述）であることが判明した場合に、その本人の同意のもとで、その陽性者と一定期間内に接触が確認された者に対し通知を行う。
> ※本仕様書では、「本人」とは本アプリが入った端末を操作する利用者を表す利用者は、スマートフォンを活用して、①日常において自らの行動変容を意識できると共に、②互いに誰とどこで接触があったのかは分からないよう、プライバシー保護と本人同意を前提に、自らが陽性者と接触した情報について、通知を受けることが可能になる。
>（利用者におけるメリット）
>・利用について本人の同意のもと、自分が感染者と接触が確認された者かどうかを知ることができる
>（公衆衛生当局、保健所等におけるメリット）
>・個人が自らの行動変容を意識するとともに、接触確認後の適切な行動等を実施できることにより、感染拡大の防止につながる

**Reply by keiji** [reply]
会計検査院法第34条の規定による処置要求及び同法第36条の規定による処置要求
https://www.jbaudit.go.jp/pr/kensa/result/3/r031027.html

> 　会計検査院は、合規性等の観点から、仕様書に業務の実施に係る事項は適切に定められているか、受注者側における業務の実施状況を十分に把握しているか、ＣＯＣＯＡを特定の端末で使用した場合に、陽性者近接状態に該当した場合であっても、接触確認者に通知が行われていない事態が発生した際の対応は適切に行われているかなどに着眼して検査しました。
>　その結果、令和３年10月27日、厚生労働大臣に対し、会計検査院法第34条の規定により是正改善の処置を求め、及び同法第36条の規定により改善の処置を要求しました。

**Reply by keiji** [reply]
Google COVID-19 Exposure Notifications Service Additional Terms
https://blog.google/documents/72/Exposure_Notifications_Service_Additional_Terms.pdf/

**Reply by keiji** [reply]
シンガポールの接触追跡アプリが方針転換、犯罪捜査でも利用可に 2021.01.14
https://www.technologyreview.jp/s/230403/singapores-police-now-have-access-to-contact-tracing-data/

> シンガポール警察は、同国の接触追跡アプリで収集したデータを犯罪捜査に利用可能にすると発表した。各国で運用されている接触追跡アプリについては、プライバシーに関する懸念が以前から指摘されており、今回のシンガポール当局の姿勢に批判が集まっている。

> スマートフォンアプリ、または小型ウェアラブル機器で利用できるトレース・トゥギャザーは、シンガポールに住む570万人の市民の80%近くが使用している。

PUBLIC DIGITALで書かれているよりも利用者数を伸ばしていた。

**Reply by keiji** [reply]
独警察、接触者確認アプリのデータを捜査利用 批判の的に 2022年1月12日
https://media.dglab.com/2022/01/12-afp-01-2/

> 【AFP＝時事】ドイツ警察は11日、新型コロナウイルス接触確認アプリのデータを捜査に利用したことで批判の的となった。
>　警察は、西部マインツ（Mainz）のレストランで昨年11月に男性が転倒・死亡した事故の目撃者を捜すため、飲食店やバーの利用履歴を登録して新型ウイルスの陽性者と接触した可能性を通知するアプリ「Luca」のデータを利用した。

ドイツでExposureNotification APIを利用しているアプリ「Corona-Warn-App」ではない別の接触確認アプリの話。

**Reply by keiji** [reply]
令和2年8月21日 事務連絡
新型コロナウイルス接触確認アプリ（COCOA）で通知を受けた者に対する行政検査等について
https://www.mhlw.go.jp/content/000661724.pdf

**Reply by keiji** [reply]
ECDC(European Centre for Disease Prevention and Control: 欧州疾病予防管理センター)と、WHO（World Health Organization: 世界保健機関）の本部・欧州地域事務局が共同開発したデジタル接触追跡の評価フレームワーク

Indicator framework to evaluate the public health effectiveness of digital proximity tracing solutions
https://www.ecdc.europa.eu/en/publications-data/indicator-framework-evaluate-public-health-effectiveness-digital-proximity

**Reply by keiji** [reply]
ExposureWindow modeへの対応に向けた検討について
https://speakerdeck.com/keiji/android-bazaar-and-conference-2021-spring
